azavea/echo-locator
# Remove references to ECC

## Overview

Remove labels, profile setting, and filtering for ECC neighborhoods.

Closes #308.


### Notes

Does not remove the ECC field from the data processing scripts; we can do that with the next data update.


## Testing Instructions

 * Profile page should no longer display option for filtering by ECC
 * ECC labels should not show on neighborhood list card or detail view
 * Results should no longer filter by ECC (even if profile was set to do so)
